### PR TITLE
Feature/devise

### PR DESCRIPTION
**Pull request by boypie510**
1.Set up gem devise
2.Add parameter [:display_name, :gender, :age] for registeration 
3.Update users column

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -28,6 +28,9 @@ gem 'jbuilder', '~> 2.7'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.4.2', require: false
 
+# Member system
+gem 'devise', '~> 4.2'
+
```
Comment (TeKaiPeng) [inline Gemfile:33]: omg

Comment (TeKaiPeng) [reply Gemfile:33]: test

**Comment by TeKaiPeng** [comment]
we are the best!

**Review by tungchinlee — APPROVED** [review]

**Comment by leegocode** [comment]
!

**Review by leegocode — APPROVED** [review]

**Pull request closed without merging**